src/useWizard.ts

```diff
@@ -12,6 +12,12 @@ const useWizard = (
   const isPrevDisabled: boolean = currentStep === 0;
   const isFirstStep: boolean = currentStep === 0;
   const isLastStep: boolean = currentStep >= total - 1;
+  useEffect(
+    () => {
+      setCurrentStep(activeStepIndex);
+    }
```

Comment:
There should be a comma here. Did you find it working in your local machine? I'm sure it didn't.